test/fixtures/integration/multiple-replace/tests.html

```diff
@@ -158,6 +165,48 @@
       assert.equal(insideElement.nodeName, 'ORIGINAL-TEMPLATE-SUCCESS');
     });
   });
+
+  describe('testing template bindings', function() {
```

Comment:
I added a test that checks if the bindings still work after the replacement. @kevinpschaaf or @sorvell do you forsee any issues that would arise regarding template bindings and instancing a copied template without certain binding identifiers?